src/main/java/frc/robot/Subsystems/Drivetrain.java

```diff
@@ -0,0 +1,148 @@
+package frc.robot.Subsystems;
+
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+import com.spikes2212.control.FeedForwardSettings;
+import com.spikes2212.control.PIDSettings;
+import com.spikes2212.dashboard.RootNamespace;
+import com.spikes2212.util.PigeonWrapper;
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.kinematics.DifferentialDriveOdometry;
+import edu.wpi.first.math.kinematics.DifferentialDriveWheelSpeeds;
+import edu.wpi.first.wpilibj.Encoder;
+import edu.wpi.first.wpilibj.motorcontrol.MotorControllerGroup;
+import edu.wpi.first.wpilibj.smartdashboard.Field2d;
+import frc.robot.RobotMap;
+
+import java.util.function.Supplier;
+
+public class Drivetrain extends OdometryDrivetrain2 {
+
+    // TODO
+    private static final double WIDTH = 0.6;
```

Comment:
put this constant on the dashboard

Reply:
Sitton told me not to
Don't blame the player, blame the game